server.js

```diff
@@ -0,0 +1,19 @@
+const http = require('http');
+
+const hostname = '0.0.0.0';
+const port = 3000;
+
+// Computes fibonacci:
+let a = 1;
+let b = 1;
+for (let i = 0; i < 10; i += 1) [a, b] = [b, a + b];
```

Comment:
here is the tenth number:
![image](https://github.com/networkflow/github-demo/assets/47474081/daed93c4-e249-44e7-b2c4-dd02311dcc0a)
and here is the result from your code:
![image](https://github.com/networkflow/github-demo/assets/47474081/a1c12299-0223-4cf8-85df-d2b15750bd44)
please check your result